laravel-api/api/User/Models/Contact.php

```diff
@@ -17,12 +17,12 @@ class Contact extends Model
      *
      * @var array
      */
-    // protected $fillable = [
-        // 'domain_uuid', 'contact_type', 'contact_nickname'
-    // ];
+    protected $fillable = [
```

Comment:
Use either fillable or guarded.

Reply:
Why do you allow users to enter domain id? It this expected?

Reply:
# Use either fillable or guarded.

Don't allow fields like `domain_uuid`, `contact_uuid` which can cause security problems.